Dockerfile

```diff
@@ -2,12 +2,14 @@ FROM ruby:2.5.1-alpine
 MAINTAINER apply for legal aid team
 
 RUN apk --update add --virtual build-dependencies \
-                               build-base \
-                               libxml2-dev \
-                               libxslt-dev \
-                               sqlite-dev \
-                               tzdata \
-                               && rm -rf /var/cache/apk/*
+                                    build-base \
+                                    libxml2-dev \
+                                    libxslt-dev \
+                                    postgresql-dev \
+#                                    sqlite-dev \
```

Comment:
Re-adds lines that was removed in #3 - see ba56a87ef2ed87a6c1e5794971eff451942927a1 and 8d043ea6d5fa07341eae65f323a846525f2c75d3